server/src/main/java/org/opensearch/index/SegmentReplicationShardStats.java

```diff
@@ -118,7 +119,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.field("current_replication_lag", new TimeValue(currentReplicationLagMillis));
         builder.field("last_completed_replication_time", new TimeValue(lastCompletedReplicationTimeMillis));
         if (currentReplicationState != null) {
-            builder.startObject();
+            builder.startObject("current_replication_state");
```

Comment:
why do we need an xcontent update here?

Reply:
This is not related to this change, I was testing something and encountered error after adding this it was resolved. But i forgot the context. I can remove this for this PR